src/components/card-with-image/card-with-image.styles.js

```diff
@@ -0,0 +1,52 @@
+export const styles = {
+  left: {
+    box: {
+      display: 'flex',
+      width: '1116px',

```

Comment:
We have this width in the container. Do we still need to use it here?

Reply:
we don't have container with styles on guestHomePage
and it's hard to do, because first block need to be full width

Reply:
Then other blocks should be wrapped, I think.

Reply:
I wrapped other blocks, but if i remove this width from my text block, my text will be displayed incorrectly and cut my container, to fix this i need to use textAlign: 'justify', but i already use textAlign:"start/end".
I think the problem is in text-align, and if i don't have fixed width, text will be displayed incorrectly.